.buildkite/pipeline.yml

```diff
@@ -1,12 +1,8 @@
 steps:
-  - command: "source ~/.cargo/env && cargo fmt --all -- --check"
-    label: "Formatter"
-    
-  - command: "source ~/.cargo/env && cargo clippy"
-    label: "Linter"
-    
-  - command: "source ~/.cargo/env && cargo test"
-    label: "Tests"
-
-  - command: "fossa init && fossa analyze --server-scan && fossa test"
-    label: "Fossa"
```

Comment:
Why do you believe it is not needed?

Reply:
@mhalambek we cannot delete this.

Reply:
I believe that @mhalambek wanted to add this as a seperate pipeline, but didnt realized there was already this config. Rename the new one to docker-release.yml and leave the original one as is.

Reply:
I've deleted this by mistake, forgot to check if there is already something there